### PR TITLE
Fk/add transformers

### PR DESCRIPTION
**Pull request by absrd**
Needed for adding prefixes to outputs for syslog parsing in units.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@connectedcars/logutil",
-  "version": "2.2.0",
+  "version": "2.3.0",
```
Comment (mex) [inline package.json]: Wait with this until after it's merged to master. Once it's merged, you can do `npm version major|minor|patch`, which will automatically update package.json and add a git tag.

File: src/config.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const postFormatInterceptors = []
+
+module.exports = {
+  _postFormatInterceptors: postFormatInterceptors,
```
Comment (mex) [inline src/config.js]: We generally don't export variables prefixed with underscore. Maybe remove the underscore and then update `src/index.js` to only pass along the `addPostFormatInterceptor` method.

File: src/log.js [hunk 3]
```diff
@@ -1,8 +1,12 @@
+const { _postFormatInterceptors } = require('./config')
 const { logLevels, getLogLevel } = require('./levels')
 const { format } = require('./format')
 
 const doLog = (level, ...args) => {
-  const output = format(level, ...args)
+  let output = format(level, ...args)
+  _postFormatInterceptors.forEach(interceptor => {
```
Comment (mex) [inline src/log.js]: Perhaps add a test case to one (or each) of the log methods just to be absolutely sure this works as expected.

**Comment by absrd** [comment]
Closing it as it is apparently not needed. Had not seen that firmware logs where interpreted in logger-cc

**Pull request closed without merging**